brave/brave-core
# Adds grant transition banner

Resolves https://github.com/brave/brave-browser/issues/7510

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
- go to release channel
- enable rewards
- put 100 + BAT into your anon wallet
- switch to this PR
- make sure that you see a banner about the transition

(do the same thing but with less than 100 BAT scenario and one where you have verified wallet connected. In both cases you should not see a banner)

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
